source/data-handling-database-R.Rmd

```diff
@@ -0,0 +1,150 @@
+---
+title: "Read data from INBO databases (SQL Server) with R(ODBC)"
+maintainer: "Stijn Van Hoey"
+date: "February 3, 2017"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+To query data from a SQL database that is already accessible using MSAccess, such a database is also accessible from R using the package [RODBC](https://cran.r-project.org/web/packages/RODBC/index.html). This package enables the link in between R and the (remote) database. After installation of the package (`install.packages('RODBC')`), the package can be loaded:
+
+```{r, loadpackage, warning = FALSE, message = FALSE}
+library(RODBC)
+```
+
+For Windows users, the most important element is to know the so-called `DSN` (i.e. a registered Data Source Name). Actually, it is just the name of the database as it is known by your computer (and MS Access). The easiest way to check the `DSN` is to check the [*registered ODBC connections*](http://www.stata.com/support/faqs/data-management/configuring-odbc-win/) in the administrator tools menu. 
+
+For Dutch-speaking Windows 7 users: 
+
+    > Kies in het Configuratiescherm van Windows de optie Systeembeheer > Gegevensbronnen (ODBC). De optie Systeembeheer verschijnt in de categorie Systeem en onderhoud.
+
+You should see a list similar to the list underneath, with the names of the available DSN names enlisted:
+![odbc-connecties](../img/odbc_gegevensbron.png)
+
+An alternative way to check the DSN name of a database already working on with Access, is to check the DSN inside MS Access (in dutch, check menu item *Koppelingsbeheer*): 
+![access-dsn](../img/access_dsn.png)
+
+For example, the DSN name `UserFlora` or `Cydonia-prd` can be used to query these databases and extract data from it with similar queries to the one used in MSAccess. First of all, the connection with the database need to be established, by using the `odbcConnect` function, providing the DSN name as argument:
+
+```{r sqlconnection}
+# WINDOWS USERS (uncomment when working in Windows)
+#my_connection <- odbcConnect("UserFlora") # uncomment for windows users
+
+# LINUX/MAC USERS (comment out when working in Windows)
+my_connection <- odbcDriverConnect("Driver={ODBC Driver 13 for SQL Server};Server=inbosql04.inbo.be;Database=UserFlora;Trusted_Connection=yes;")
```

Comment:
I agree, whereas this is indeed out of scope for this tutorial, it should be provided elsewhere. I'll create a separate document for linux-users in the near future, as this requires an ODBC-driver and kerberos utilities for linux-users. 

I focused with this tutorial on windows-users with an existing ODBC connection.